contracts/validators/CredentialAtomicQueryValidator.sol

```diff
@@ -122,4 +157,19 @@ abstract contract CredentialAtomicQueryValidator is OwnableUpgradeable, ICircuit
             revert("Generated proof is outdated");
```

Comment:
this check is tricky ` if (block.timestamp - _proofGenerationTimestamp > proofGenerationExpirationTime)` witll give true if _proofGenerationTimestamp > block.timestamp (future proof) , because it will be a huge number and it will be bigger than 1 hour